src/main/java/org/broadinstitute/hellbender/tools/gvs/filtering/CreateFilteringFiles.java

```diff
@@ -37,10 +37,10 @@ public final class CreateFilteringFiles extends VariantWalker {
     private SimpleXSVWriter writer;
 
     private List<String> HEADER = 
-        Arrays.asList("filter_set_name", "mode", "location", "ref", "alt", "vqslod", "culprit","training_label","yng");
+        Arrays.asList("filter_set_name", "mode", "location", "ref", "alt", "vqslod", "culprit", "training_label", "yng");
     // TODO - not sure about culprit and training label.
     private List<String> HEADER_VQSR_LITE =
-        Arrays.asList("filter_set_name", "mode", "location", "ref", "alt", "calibration_sensitivity", "culprit", "training_label", "yng");
+        Arrays.asList("filter_set_name", "mode", "location", "ref", "alt", "calibration_sensitivity", "score", "training_label", "yng");
```

Comment:
oh thank god--"culprit" is so confusing

Reply:
Actually, it's gone in LITE - there isn't even a culprit provided. The 'score' is analogous to VQS_LOD (as generated by VQSR classic)